src/Admin/Models/OrganizationViewModel.cs

```diff
@@ -37,6 +40,9 @@ public class OrganizationViewModel
             orgUsers
             .Where(u => u.Type == OrganizationUserType.Admin && u.Status == organizationUserStatus)
             .Select(u => u.Email));
+        Secrets = org.UseSecretsManager ? secrets.Count().ToString() : NotApplicable;
+        Projects = org.UseSecretsManager ? projects.Count().ToString() : NotApplicable;
+        ServiceAccounts = org.UseSecretsManager ? serviceAccounts.Count().ToString() : NotApplicable;
```

Comment:
Seems a bit weird that we're now storing numbers as strings here. I think it's better to have this logic in the template, e.g.: https://github.com/bitwarden/server/blob/d35de15dfa444ec0af56f3027b3f4ff49f41c96b/src/Admin/Views/Organizations/_ViewInformation.cshtml#L20-L22

so that these properties can just be `int`s.

Reply:
resolved